js/date-filter.js

```diff
@@ -25,8 +25,13 @@ function DateFilter(elm) {
     oncomplete: this.validate.bind(this)
   });
 
+  if (this.$minDate.attr('data-min-date')) {
```

Comment:
Is this necessary? I don't think we should be setting min and max dates by default here. It's ok to have them empty.